ui/stage/stage-test.tsx

```diff
@@ -58,11 +60,14 @@ export function StageTest({
   const testData = useTestData(stage);
   const inputState = useInputState(stage);
 
-  return (
-    <div className="pad">
-      {timez(9, (idx) => (
-        <FsrPanel active={inputState?.[idx]} key={idx} testData={testData?.panels[idx]} />
-      ))}
-    </div>
-  );
+  const useFSRs = false;
+  (stage?.info?.firmware_version || 5) >= 5;
```

Comment:
If you want to check if a specific stage is FSR or not you can check `stage.config.config.flags.PlatformFlags_FSR` which will be true if it's an FSR stage. I think at some point I might need to make some of these names cleaner lol.